pkg/report/cyclonedx/cyclonedx.go

```diff
@@ -31,12 +32,22 @@ func (w Writer) Write(report types.Report) error {
 	if err != nil {
 		return xerrors.Errorf("CycloneDX marshal error: %w", err)
 	}
+	return w.encodeBom(bom)
+}
 
+func (w Writer) WriteKbom(report rep.Report) error {
```

Comment:
It is kubernetes-specific. I think it should be located under `pkg/k8s/report`.

Reply:
we might need to add this function instead :
```
func (w Writer) WriteComponent(component core.Component) {
	bom:=w.marshaler.core.Marshal(component)
	return encodeBom(bom)
}
```
the other option is to expose `marshaler` and `Marshaler.core`
what do you prefer?